src/index.js

```diff
@@ -15,13 +15,16 @@ const sortDeep = (object) => {
 
 module.exports = (chai, utils) => {
   const { Assertion } = chai;
-  utils.addMethod(Assertion.prototype, 'equalInAnyOrder', function equalInAnyOrder(b) {
+  utils.addMethod(Assertion.prototype, 'equalInAnyOrder', function equalInAnyOrder(b, m) {
     const a = this.__flags.object;
-    const { negate } = this.__flags;
+    const { negate, message } = this.__flags;
+
+    const msg = m || message;
+
     if (negate) {
-      new Assertion(sortDeep(a)).to.not.deep.equal(sortDeep(b));
+      new Assertion(sortDeep(a), msg).to.not.deep.equal(sortDeep(b));
```

Comment:
Thanks for this PR - could you cover this feature with some tests?

Reply:
Added tests that checking that custom message is actually prepended.

I'm not sure about messages checking, because if I will compare with whole `message: expected true to deeply equal false` it can be broken if somewhy/somewhen messages formatting will be changed. How to better test it?